service/src/test/resources/ehr/mapper/observation/expected-output-observation-statement-16.xml

```diff
@@ -0,0 +1,25 @@
+<component typeCode="COMP" >
+    <ObservationStatement classCode="OBS" moodCode="EVN">
+        <id root="394559384658936" />
+        <statusCode code="COMPLETE" />
+        <effectiveTime>
+            <center value="20190601113000"/>
+        </effectiveTime>
+        <availabilityTime value="20190601113000"/>
+        
+        <pertinentInformation typeCode="PERT">
+            <sequenceNumber value="+1" />
+            <pertinentAnnotation classCode="OBS" moodCode="EVN">
+                <text>Test comment</text>
+            </pertinentAnnotation>
+        </pertinentInformation>
+        
+        <code nullFlavor="UNK"><originalText>Mocked code</originalText></code>
+        
+        <Participant typeCode="PRF" contextControlCode="OP">
+    <agentRef classCode="AGNT">
+        <id root="394559384658936"/>
+    </agentRef>
+</Participant>
```

Comment:
indentation needs to be corrected

Reply:
Only possible if we start to ignore whitespace for the relevant tests